dist.sh

```diff
@@ -2,38 +2,32 @@
 
 set -o errexit
 
-if [[ -z ${BINARY} ]] || [[ -z ${VERSION} ]]; then
-	echo "Missing required env var: BINARY=X VERSION=X $(basename $0)"
-	exit 1
+BINARY=${BINARY:-}
+VERSION=${VERSION:-}
+
+if [[ -z "$BINARY" ]] || [[ -z "$VERSION" ]]; then
+  echo "Missing required environment variable: BINARY and VERSION"
+  echo "Usage: BINARY=<binary-name> VERSION=<version-number> $0"
+  exit 1
 fi
 
 ARCHS=(darwin-amd64 linux-amd64 linux-arm64 linux-ppc64le linux-armv6 freebsd-amd64 windows-amd64)
+RELEASE_DIR="release"
+mkdir -p "$RELEASE_DIR"
```

Comment:
Can we be consistent about a choice of `$VAR` or `${VAR}`, I tend to prefer with the braces. There's a mix in this file currently